src/04-modality-specific-files/02-magnetoencephalography.md

```diff
@@ -588,26 +588,19 @@ has to be updated, then for MEG it could be considered to be a new session.
 
 ## Empty-room MEG recordings
 
-Empty-room MEG recordings capture the environmental and recording system's
-noise.
-In the context of BIDS it is RECOMMENDED to perform an empty-room recording for
-each experimental session.
-It is RECOMMENDED to store the empty-room recording inside a subject directory
-named `sub-emptyroom`.
-The label for the [`task-<label>`](../99-appendices/09-entities.md#task) entity in the empty-room recording SHOULD be
-set to `noise`.
-If a [`session-<label>`](../99-appendices/09-entities.md#ses) entity is present, its label SHOULD be the date of the
-empty-room recording in the format `YYYYMMDD`, that is `ses-YYYYMMDD`.
-The `scans.tsv` file containing the date and time of the acquisition SHOULD
-also be included.
-The rationale is that this naming scheme will allow users to easily retrieve the
-empty-room recording that best matches a particular experimental session, based
-on date and time of the recording.
-It should be possible to query empty-room recordings just like usual subject
-recordings, hence all metadata sidecar files (such as the `channels.tsv`) file
-SHOULD be present as well.
+Empty-room MEG recordings capture the environmental and recording system's noise.
+In the context of BIDS it is RECOMMENDED to perform an empty-room recording for each experimental session.
```

Comment:
There are two recommendations, one on the lab practice and one on the data organization, and the two are not really aligned. These came out of [BEP008](https://bids.neuroimaging.io/bep008), see page 15, which happened to be the first big BEP that was merged. It might have been better had it been captured in 2018 as not being optimal.

The two practices are both being used, and I personally recommend that everyone does in their lab whatever works for them. I think it would be better if BIDS would restrict itself to data organization, and not to lab procedures. But simply scrapping the "empty-room recording per session" from the spec might lead to the impression that that is not recommended, whereas it actually was the recommended strategy (by consensus) in BEP006.

Should we rephrase, say that both are possible, that "empty-room recording per session" is recommended (to stay in line with BEP006) and give an example for both? Actually, compared to the current text in this PR the only change would then be to also give an example for the "empty-room recording per session".

Reply:
I agree with @robertoostenveld.  Still, how would you feel about changing a single word:  "In the context of BIDS it is RECOMMENDED to **associate** an empty-room recording for each experimental session."  This implicitly maintains the idea that it should be performed, while bringing it back to the organization context of BIDS, and the statement now works for both practices.